DataFormats/MuonReco/interface/Muon.h

```diff
@@ -219,7 +219,11 @@ namespace reco {
       MultiIsoMedium         = 1UL<<26,   // miniIso with ptRatio and ptRel 
       PuppiIsoLoose          = 1UL<<27,
       PuppiIsoMedium         = 1UL<<28,
-      PuppiIsoTight          = 1UL<<29
+      PuppiIsoTight          = 1UL<<29, 
+      MvaVTight              = 1UL<<30,   // > 0.45
+      MvaVVTight             = 1UL<<31,   // > 0.9
```

Comment:
from https://github.com/cms-sw/cmssw/pull/26460/files#r276415430
> I think that there is little value to specify the values of the cuts in the header of reco::Muon. These are not fundamental constants and will likely become obsolete in the next update.

I see that there was an update as requested
https://github.com/cms-sw/cmssw/pull/26460/commits/3b61ce1c5b27b21098c76a2a4a2b35a60da4a283
but then it was reverted with a commit that has "removing comments":
https://github.com/cms-sw/cmssw/pull/26460/commits/4114cf871d188ec059e67d4221097886417ce1bc
What happened?

Reply:
i'm not sure what happened, it was unintended. i'm fixing it